### DISCUSSION TITLE
Tutorial 3: Black Litterman Mean Risk Optimization

### DISCUSSION THREAD

**Discussion by FeCol150** [dcajasn/Riskfolio-Lib#166] [Q&A]
Hi,

According to the tutorial 3. The way you get the BL model is "Proportional to the variance: We can just assume that the variance of the views will be proportional  to the variance of the asset return, just as the variance of the prior distrubiton is." as it says in Black Litterman in detail, right?

Best,
Fabio

**Comment by heykav** [comment]
Yes, that's exactly right - and you can see it directly in the source rather than just the tutorial prose. In `black_litterman()` (`riskfolio/src/ParamsEstimation.py`, ~line 1240):

```python
tau = 1 / X.shape[0]
Omega = np.array(np.diag(np.diag(P @ (tau * S) @ P.T)), ndmin=2)
```

That's literally `diag(P (\u03c4\u03a3) P')` - the classic He & Litterman / Idzorek default: each view's uncertainty variance is set proportional to that view portfolio's own variance under the (scaled) prior covariance, with off-diagonal terms dropped (view errors assumed uncorrelated). So the variance of the *view* really is being set proportional to the variance of the *prior*, exactly as Walters describes in *Black-Litterman in Detail* - Riskfolio-Lib isn't doing anything more exotic than that by default.

One detail worth knowing alongside this: `tau` itself isn't a parameter you pass in here - it's hardcoded as `1/T` (number of observations in your returns sample), which is the common textbook default but means you can't currently override it directly through this function if you wanted to experiment with a different `tau`.
